Tabloid/client/src/components/ApplicationViews.js

```diff
@@ -3,6 +3,7 @@ import { Switch, Route, Redirect } from "react-router-dom";
 import Login from "./Login";
 import Register from "./Register";
 import Hello from "./Hello";
+import PostList from "./Posts/PostList";
```

Comment:
Posts in directory name does not need to be capitalized

Reply:
Fixed!